lib/sysrev/sysrevMeta.js

```diff
@@ -129,12 +130,20 @@ module.exports = function(options) {
     var re = new RegExp('^' + identifier + '.*', ['i']);
```

Comment:
Maybe we can just drop the support for partial match? Seems like it will just be trouble.

It also simplifies the regex pattern to `'^' + identifier + '$'` and removes the need for:
```
 +    // find exact match on name
 +    if (!systemId) {
 +      systemId = _.find(_.keys(_systems), function(system) { return _systems[system].name === identifier; });
 +    }
 +
```

Reply:
I could agree with that. To me a partial match should be CLI only with a prompt (ie. "Did you mean Systemname? (y/n):")